CONTRIBUTORS.rst

```diff
@@ -176,6 +176,7 @@ Listed in alphabetical order.
   Oleg Russkin             `@rolep`_
   Pablo                    `@oubiga`_
   Parbhat Puri             `@parbhat`_
+  Pawan Chaurasia          `@rjsnh1522`_
```

Comment:
That's currently a broken rst link, the target should be inserted around line ~348.

Reply:
fixed.